onnxconverter_common/onnx_ops.py

```diff
@@ -850,6 +850,10 @@ def apply_sigmoid(scope, input_name, output_name, container, operator_name=None)
     _apply_unary_operation(scope, 'Sigmoid', input_name, output_name, container, operator_name)
 
 
+def apply_softsign(scope, input_name, output_name, container, operator_name=None):
+    _apply_unary_operation(scope, 'Softsign', input_name, output_name, container, operator_name)
```

Comment:
Thanks for the contribution!
`_appy_unary_operation` will set `op_version=6` for `Softsign` if `target_opset>=6`. However, `Softsign` only has opset 1. So please just use `op_version=1` here,

Reply:
that makes sense, I've updated the code to always create an opset 1 node.